src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/IntegrationTest.kt

```diff
@@ -45,6 +45,13 @@ abstract class IntegrationTest {
     @JvmField
     internal val caseNotesMockServer = CaseNotesMockServer()
 
+    private var lastBookingId = 0L
+
+    fun getNextBookingId(): Long {
```

Comment:
Not the best but works for now. I want to move all none end to end type tests into controller tests going forward in the hope to reduce complexity.